source/includes/connection-snippets/x509-snippets/nodejs-connection.js

```diff
@@ -0,0 +1,20 @@
+// begin x509 connection
+const MongoClient = require('mongodb').MongoClient;
```

Comment:
per the style above, here's what this looks like now:

```
const { MongoClient } = require('mongodb');
const fs = require('fs');

const client = new MongoClient('mongodb+srv://<cluster-url>?authMechanism=MONGODB-X509&ssl=true', {
  sslCA: fs.readFileSync('/etc/certs/mongodb/ca.pem'),
  sslKey: fs.readFileSync('/etc/certs/mongodb/client.pem'),
  sslCert: fs.readFileSync('/etc/certs/mongodb/client.pem')
});

async function run() {
  try {
    await client.connect();
    const collection = client.db('test').collection('devices');
    // perform actions on the collection object
  } finally {
    // Ensures that the client will close when you finish/error
    await client.close();
  }
}

run().catch(console.dir);
```

Reply:
Fixed - replaced my node snippet almost verbatim with the one you provided above ^ the only thing I changed was the removal of the line ` const collection = client.db('test').collection('devices');` to make it consistent w/ the other examples in this repo. This ticket only entails client connection, not specifying a db or collection. Let me know if that's fine with you @mbroadst